skimage/transform/hough_transform.py

```diff
@@ -17,15 +17,15 @@ def hough_line(img, theta=None):
         Input image with nonzero values representing edges.
     theta : 1D ndarray of double
         Angles at which to compute the transform, in radians.
-        Defaults to -pi/2 .. pi/2
+        Defaults to -pi/2 .. pi/2 with 180 values.
 
     Returns
     -------
-    H : 2-D ndarray of uint64
+    hspace : 2-D ndarray of uint64
```

Comment:
I'm sorry, but I personally don't care to be consistent to their signatures and not at the price to be inconsistent with ourself :) Why them and not others? I never used matlab and while they are doing non foss softwares, I'm not going to pay attention to what they do. In addition, @stefanv recently said that to build something instead of reproducing (cf recent video posted the mailing list). I think it's a solid point if we want to make progress